openhands/runtime/plugins/agent_skills/file_ops/file_ops.py

```diff
@@ -370,6 +373,24 @@ def find_file(file_name: str, dir_path: str = './') -> None:
         print(f'[No matches found for "{file_name}" in {dir_path}]')
 
 
+def get_repomap(repo_path: str = './', message_history='') -> None:
```

Comment:
**Suggestion:** Use a more descriptive name for the `get_repomap` function.

**Existing Code:**
```Python
def get_repomap(repo_path: str = './', message_history='') -> None:
```

**Improved Code:**
```Python
def retrieve_repo_map(repo_path: str = './', message_history='') -> None:
```

**Details:**
Use a more descriptive name for the `get_repomap` function to clarify its purpose.